change profile picture

rgdeekshith opened the pull request:
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

Mohammed-Shoaib commented on _data/rgdeekshith/rgdeekshith.md:
```diff
@@ -0,0 +1,7 @@
+---
+name: Rallapalli Goutham Deekshith
+institution: Hyderabad Institute of Technology and Management
+profile_pic: rgdeekshith.jpg
```
Incorrect file name, please do one of the following:
1. Change `rgdeekshith.jpg` to `rgdeekshith.png` in the markdown file.
2. Rename `rgdeekshith.png` to `rgdeekshith.jpg`.
I'd suggest you make the change in the markdown file!

rgdeekshith commented:
> Hey @rgdeekshith,
> 
> Congrats on graduating! :mortar_board: :tada:
> 
> It seems like you are pointing to an image file that doesn't exist because the file extension is wrong. Could you please fix this? :memo:
> 
> Thanks,
> Mohammed Shoaib

I didn't understand sir, could you elaborate!

rgdeekshith commented:
> 
> 
> Hey @rgdeekshith,
> 
> Congrats on graduating! 🎓 🎉
> 
> It seems like you are pointing to an image file that doesn't exist because the file extension is wrong. Could you please fix this? 📝
> 
> Thanks,
> Mohammed Shoaib

Sir, I fixed the issue which you mentioned and i got a conflict, further i resolved please check if that's correct, I'm new to this PR and commits, please help me if its wrong.

rgdeekshith commented:
> 
> 
> Hey @rgdeekshith,
> 
> Thank you for changing the file extension in the markdown file!
> 
> However, it seems like you added a couple of extra lines in your last 2 commits. You need to follow the format mentioned in the instructions. Please read instructions on the [README](https://github.com/education/graduation/blob/master/README.md) file for more information.
> 
> Below I've shown how your markdown file should look like:
> 
> ```
> ---
> name: Rallapalli Goutham Deekshith
> institution: Hyderabad Institute of Technology and Management
> profile_pic: rgdeekshith.png
> quote: It's not over until it's over.
> github_user: rgdeekshith
> ---
> ```
> 
> Could you please paste **only** the above in your markdown file? 📝
> 
> Thanks,
> Mohammed Shoaib

The changes are made, please review.

Mohammed-Shoaib commented:
Hey @rgdeekshith,

Seems like your info had already been merged previously. I assume you are making this PR to add a different image of yourself. Hence, you'll have to delete the old image from your folder. Kindly delete `rgdeekshith.jpg`, i.e., the old image that had been merged previously.

Thanks,
Mohammed Shoaib

rgdeekshith commented:
> 
> 
> Hey @rgdeekshith,
> 
> Seems like your info had already been merged previously. I assume you are making this PR to add a different image of yourself. Hence, you'll have to delete the old image from your folder. Kindly delete `rgdeekshith.jpg`, i.e., the old image that had been merged previously.
> 
> Thanks,
> Mohammed Shoaib

Sir, old image `rgdeekshith.jpg` deleted.

rgdeekshith commented:
> 
> 
> Hey @rgdeekshith,
> 
> Thanks for deleting the old image! I'd suggest you change the title of your PR to something more relevant such as "Change profile picture".
> 
> Your pull request looks good 👌
> 
> I confirm the following is correct:
> 
>     1. Checklist is complete. ✔️
> 
>     2. Pull request only modified submission files (2 files) 📂
> 
>     3. Profile picture is squared and has at least 544 pixels 🔳
> 
> 
> I wish you all the best in your future endeavors! 🦄
> 
> Best,
> Mohammed Shoaib

Thank You Sir,
I don't know how to change the PR, please confirm that the present name of the PR is alright for now, but here I've been shown messages like `Review required` & `Merging is blocked`, both in red coloured letters, is that okay? or do I need to make any further changes?

Mohammed-Shoaib commented:
@rgdeekshith I confirm your pull request is perfect! :confetti_ball: I approved the changes. :+1:
It's very easy to change the title, scroll all the way up, and rename `Grad1` to something more relevant if you can!

rgdeekshith commented:
> 
> 
> @rgdeekshith I confirm your pull request is perfect! 🎊 I approved the changes. 👍
> It's very easy to change the title, scroll all the way up, and rename `Grad1` to something more relevant if you can!

All changes made. Thank You Sir!

rgdeekshith commented:
Sir, but the previous reviewer told that everything thing is fine and can continue and done changes as he insisted.

rgdeekshith commented:
> We have reviewed 📝 your application to the yearbook. It looks great ✨ but we found that You are modifying 3 files while you should only modify 2 (one image and one md file). I will close this PR so you can do the changes requested. You can open a new one once you've done the fixes and we will review it again.😊

Sir, the previous reviewer told me to do that changes and I have done. I followed the instructions as he insisted me. I removed one of the files as I replaced it with a new file and the previous reviewer also agreed with it.
